modules/dms-endpoint/variables.tf

```diff
@@ -120,7 +120,7 @@ variable "postgres_settings" {
     heartbeat_schema             = optional(string, false)
     map_boolean_as_boolean       = optional(bool, false)
     map_jsonb_as_clob            = optional(bool, false)
-    map_long_varchar_as          = optional(bool, false)
+    map_long_varchar_as          = optional(string, "wstring")
```

Comment:
@marsadle could you expand on why `"wstring"` is a default value here?

Reply:
As mentioned in the PR description, this is the default value according to [AWS documentation](https://docs.aws.amazon.com/dms/latest/APIReference/API_PostgreSQLSettings.html#DMS-Type-PostgreSQLSettings-MapLongVarcharAs).